pkg/grpc/README.md

```diff
@@ -12,20 +12,6 @@ go get -u github.com/golang/protobuf/protoc-gen-go
 
 ## Package structure
 
-* `grpc/proto/` - a copy of proto files from [protobuf-schemas](https://github.com/wavesplatform/protobuf-schemas) project. Files are copied from folders `proto/waves/` and `proto/waves/node/grpc`. And `import` directives updated afterwards to reflect the flat structure.
+* `grpc/protobuf-schemas/` - a submodule of [protobuf-schemas](https://github.com/wavesplatform/protobuf-schemas) project (proto files).
 * `grpc/generated` - code generated from proto files.
 * `grpc/server` - gRPC server implementation (API).
```

Comment:
It will be useful to put here information how to update submodule and regenerate protobuf code.